src/systemtest/java/reposense/ConfigSystemTest.java

```diff
@@ -41,26 +41,42 @@ public void setUp() throws IOException {
         FileUtil.deleteDirectory(FT_TEMP_DIR);
     }
 
+    /**
+     * System test with a specified until date and a wildcard since date (capture from the very first commit).
```

Comment:
First mention of `wildcard`,  another one in the pr message.

Reply:
I feel that specifying a certain character in `--since` flag is better than just specifying `--since` flag and nothing gets parsed in, as it's best to explicitly capture the user's intended date range (or any other critical settings).

Can call it `asterisk` if it's more appropriate.

Reply:
Sounds good.

Reply:
I don't think naming `*` is useful. It may cause confusion. i.e. You have a bracket clarification.

`System test with a specified until date and a * since date to capture from the first commit`?

Reply:
Ok, will remove the naming convention of `*`